pkg/sink/tui/container/overall_container.go

```diff
@@ -105,6 +107,12 @@ func OverallVisuals(ctx context.Context, cli *client.Client, all bool, dataChann
 			errorBox.Resize(w, h)
 			ui.Render(errorBox)
 
+		case "ACTION":
+			actions.SetRect(0, 0, w/6, h)
+			page.Grid.SetRect(w/6, 0, w, h)
+			ui.Render(actions)
+			ui.Render(page.Grid)
+
```

Comment:
Maybe consider making `utilitySelected` an enum? (in the same way it was done for [`Command`](https://github.com/pesos/grofer/blob/20010bd2c7e8eca598adb14d122cbbfcec9593b5/pkg/core/types.go#L19-L33)).

Since it's run on every UI update, it would prevent many string comparisons from taking place every tick.

Is this right, or does the go compiler optimize these things?

Reply:
This does make sense, I'll update for all pages by next commit.

Reply:
Done.